catalyst-gateway/tests/api_tests/scripts/prepare_cardano_asset.py

```diff
@@ -0,0 +1,173 @@
+"""
+This script is a simple web scraper tool to prepare testing data for the `cardano/asset` endpoint.
+
+Prerequisites before running this script:
+- Make sure that you have `BeautifulSoup` installed, else install with `pip install beautifulsoup4`.
```

Comment:
Just make this script as a part of the poetry python project and add `BeautifulSoup` as a dependency inside `pyproject.toml`.
So it will be possible to run it like this
```
poetry run python3 scripts/prepare_cardano_asset.py
```